src/installer/BeatPackageCompiler/BeatPackageCompiler.cs

```diff
@@ -100,76 +105,53 @@ static void Main(string[] args)
                     System.IO.File.ReadAllText(
                         Path.Combine(opts.PackageInDir, MagicStrings.Files.LicenseTxt))));
 
-            var beatConfigPath = "[CommonAppDataFolder]" + Path.Combine(companyName, productSetName, ap.CanonicalTargetName);
-            var beatDataPath = Path.Combine(beatConfigPath, "data");
-            var beatLogsPath = Path.Combine(beatConfigPath, "logs");
+            // Add a note to the final screen and a checkbox to open the directory of .example.yml file
+            // "-" is not a legal character in wxs component names, must replace with "_"
+            var beatConfigExampleFileName = ap.CanonicalTargetName.Replace("-", "_") + ".example" + MagicStrings.Ext.DotYml;
+            var beatConfigExampleFileId = beatConfigExampleFileName + "_" + (uint) beatConfigExampleFileName.GetHashCode32();
 
-            var textInfo = new CultureInfo("en-US", false).TextInfo;
-            var serviceDisplayName = $"{companyName} {textInfo.ToTitleCase(ap.TargetName)} {ap.SemVer}";
+            var beatsInstallPath =
+                $"[ProgramFiles{(ap.Is64Bit ? "64" : string.Empty)}Folder]" +
```

Comment:
I think we only support 64-bit windows so this check is likely unnecessary?

Reply:
i can change this ofcourse... but note it's the original code that is also shared with the beats MSI, i tried no to change any logic for the beats MSI.